Taskfile.R.yaml

```diff
@@ -26,7 +26,7 @@ tasks:
     cmds:
       - task r:base:copy-files IDENTIFIER=base NBL_LANGUAGE_VERSION={{.NBL_R_VERSION}}
       # build the base image
-      - task build LANGUAGE=r NBL_LANGUAGE_VERSION={{.NBL_R_VERSION}} IDENTIFIER=base -- --build-arg NBL_R_VERSION={{.NBL_R_VERSION}}
+      - task build LANGUAGE=r NBL_LANGUAGE_VERSION={{.NBL_R_VERSION}} IDENTIFIER=base BUILD_TARGET=base -- --build-arg NBL_R_VERSION={{.NBL_R_VERSION}}
```

Comment:
this was stuck looking at `BUILD_TARGET=main` for some reason; should be cleared up in https://github.com/noteable-io/kernels/pull/59 and only affects local building here